blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -252,8 +252,101 @@ namespace blueprint
             return result;
         }
 
+        struct TimeoutFunctionManager : private juce::MultiTimer
+        {
+            ~TimeoutFunctionManager() override {
+                for(const auto &[id, timer] : timeoutFunctions)
```

Comment:
I know we're trying to keep things trim, but much like what was fixed in https://github.com/nick-thompson/blueprint/pull/182, I fear we do need at least this artefact from the slated `reset` method.